src/main/java/Assistinator/AssitinatorExceptions.java

```diff
@@ -0,0 +1,7 @@
+package Assistinator;
+
+public class AssitinatorExceptions extends Exception{
```

Comment:
Typo with Assitnator instead of Assistinator, might want to consider changing class to AssistinatorException instead of Exceptions as there is only 1 exception